articlequality/extractors/ptwiki.py

```diff
@@ -36,7 +34,13 @@ def extract_label(template):
             label = template.get(1).value
         except ValueError:
             label = None
-    return normalize_label(label)
+    if label != "?":
```

Comment:
Do we need a special case for the '?', given that `normalize_label` already replaces anything which is not one of `POSSIBLE_LABELS.keys()` by `None` ?

Reply:
We output a ton of warnings if I don't explicitly special case it.  I want to output a warning when we see something surprising and "?" is not surprising.

Reply:
There are many cases (~4k) where users just copied the example from the template documentation and then forgot to replace the description "qualidade" by an actual quality. And there are something like 10 pages where they copied some example which used `"x"` as the "quality".